sycl/tools/sycl-vxx/bin/sycl_vxx_post_link.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+"""sycl_vxx_post_link
+This is a sycl-post-link replacement which is able to parse xclbins
```

Comment:
```suggestion
This is a sycl-post-link replacement which is able to parse xclbins
```
replacement? implementation? equivalent? complement?
It does not replace, I guess, when a GPU is used.